### PR TITLE
Paul/webpack setup

### PR DESCRIPTION
**Pull request by prheee**
package.json and webpack added. PLEASE REVIEW OUR FILES!

### CODE REVIEW

**Review by courtneywkwong — APPROVED** [review]
i see the client and build!

File: package.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "scratch",
```
Comment (xak200) [inline package.json]: are we naming it scratch or scrum?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  "version": "1.0.0",
+  "description": "<<<<<<< HEAD courtney making a change/test",
```
Comment (xak200) [inline package.json]: do we want this to be the description?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon server/server.js",
+    "build": "NODE_ENV=production; webpack",
+    "dev": "NODE_ENV=development; webpack-dev-server --open | node server/server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wunderpus123/scratch.git"
```
Comment (xak200) [inline package.json]: i think this is git+https://github.com/wunderpus123/scrum.git now

File: package.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/wunderpus123/scratch/issues"
```
Comment (xak200) [inline package.json]: scrum instead of scratch

File: package.json [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  },
+  "homepage": "https://github.com/wunderpus123/scratch#readme",
```
Comment (xak200) [inline package.json]: scrum instead of scratch

**Comment by prheee** [comment]
making updates

**Comment by prheee** [comment]
updated name/description/url!

**Review by xak200 — APPROVED** [review]

**Review by Natia94 — APPROVED** [review]

**Pull request merged**